### PR TITLE
Added checkpointing and model logging to wandb

### PR DESCRIPTION
**Pull request by deanp70**

### CODE REVIEW

File: reformer_fastai/expscript.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def run_exp(task:Param(help="Task options: 'synt','lm_base','lm_rev',lm_shared_q
     torch.cuda.set_device(cuda_id)
 
     # Callbacks used for training
-    cbs = []
+    cbs = [SaveModelCallback(every_epoch=True)]
```
Comment (morganmcg1) [inline reformer_fastai/expscript.py]: I would wrap this in:
```
cbs = []
if save_model: cbs.append(SaveModelCallback(every_epoch=True))
```

File: reformer_fastai/expscript.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def init_wandb(cbs:list=[], wandb_name:str='', wandb_group:str='', wandb_notes:s
     except Exception as e:
         print(e)
 
-    cbs.append(WandbCallback(log_model=False, log_preds=False))
+    cbs.append(WandbCallback(log_model=True, log_preds=False))
```
Comment (morganmcg1) [inline reformer_fastai/expscript.py]: Can we set `log_model = save_model` instead?

**Review by morganmcg1 — COMMENTED** [review]
Couple of nits

File: reformer_fastai/expscript.py (hunk 1 above)
Comment (deanp70) [reply reformer_fastai/expscript.py]: done

File: reformer_fastai/expscript.py (hunk 2 above)
Comment (deanp70) [reply reformer_fastai/expscript.py]: done

**Pull request merged**